src/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { ToggleStyled } from './Toggle.styles';
+import { ToggleProps } from './types';
+
+export const Toggle = ({ size = 'medium', onChange, ...rest }: ToggleProps) => (
+  <ToggleStyled $size={size}>
+    <input type="checkbox" onChange={onChange} readOnly={!onChange} {...rest} />
```

Comment:
Is readOnly prop neeeded? I guess it can be controlled from the consumer like the rest of props. What do you think?

Reply:
I added it because react complains when no onChange is set. but we can make this component a little bit dumber and let the dev control it from outside.

Reply:
Yes, maybe we can let the dev decide over those props, and end up with the following simple approach, like we did in checkbox and radio:
```typescript
<input {...rest} type="checkbox" />
```

Reply:
Changed to the simple aproach. Way cleaner.